guides/common/assembly_configuring-puppet-in-satellite.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context-of-assembly_configuring-puppet-in-satellite: {context}]
+
+ifndef::context[]
+[id="assembly_configuring-puppet-in-satellite"]
+endif::[]
+ifdef::context[]
+[id="assembly_configuring-puppet-in-satellite_{context}"]
+endif::[]
+= Configuring Puppet in {Project}
+
+:context: assembly_configuring-puppet-in-satellite
+
+[role="_abstract"]
+Puppet is an optional component in {ProjectX}.
+If you installed {Project} afresh and you want to continue using Puppet, you have to enable it in your {Project}.
+If you upgraded your {Project} and you want to stop using Puppet, you can disable it and purge Puppet data and content, respectively.
+
+include::modules/proc_enabling-puppet.adoc[leveloffset=+1]
+
+ifdef::satellite[]
+[role="_additional-resources"]
+== Additional resources
+* See also link:{PuppetDocURL}[Puppet Guide] for details about management of Puppet modules and configuration management with Puppet.
```

Comment:
Hey @Lennonka 

We **try** to conform to IBM Style for references... so it would be like: 

For more information about configuration management with Puppet, see [link]()